CodeToReview.java

```diff
@@ -0,0 +1,262 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+
+
+   O | O | O 
+  -----------
+   O | O | O 
+  -----------
+   O | O | O 
+     
+
+ */
+package tictactoe;
+
+import java.util.Scanner;
+
+public class TicTacToe {
+
+    /**
+     * @param args the command line arguments
+     * @throws java.lang.InterruptedException
+     */
+    public static void main(String[] args) throws InterruptedException {
```

Comment:
This method is too big, which makes it quite challenging to follow. I suggest breaking this up into smaller methods!